types/artifacts.d.ts

```diff
@@ -461,6 +463,14 @@ declare global {
         clientRects: Rect[];
       }
 
+      export interface TraceNode {
+        metricTag: string;
+        selector: string;
+        nodeLabel?: string;
+        nodePath: string;
```

Comment:
`devtoolsNodePath` or `path` is what we use in the artifacts.  so let's go with `devtoolsNodePath`

Reply:
the rest of these names have consistent names in the artifacts and we're all good.